config/nginx.conf

```diff
@@ -97,5 +97,21 @@ http {
 
       proxy_pass http://validator_service:4567/;
     }
+
```

Comment:
Same comment as above, you may need to comment out the code for the /validator and /validatorapi/ but I would double check